### PR TITLE
[WDFT-AMS-102020] - Maya Ausset

### PR DESCRIPTION
**Pull request by MayaAusset**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,14 +5,124 @@
...
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+  
+
+    <section class="section-1">
+
+
+      <header>
+
+
```
Comment (wax-bear) [inline index.html:19]: Are these blank lines of any use? If not, I would remove them. This applies to all unnecessary blank lines in the document.

File: index.html [hunk 2]
```diff
@@ -5,14 +5,124 @@
...
+          <img src="images/spotify-logo.png" alt="logo-spotify">
+
+          <ul class="1st-UL">
+            <li><a href=""></a>Premium</li>
+            <li><a href="">Discover</a></li>
+            <li><a href="">Help</a></li>
+            <li><a href="">Download</a></li>
+          </ul>
+
+        </nav>
```
Comment (wax-bear) [inline index.html:33]: Generally the indentation looks good.

File: styles/style.css [hunk 3]
```diff
@@ -6,3 +6,183 @@ Green: #00B172
...
+    margin-top: 100px;
+    position: relative;
+    
+}
+#text-section-1{
+    font-size: 2rem;
+    color: white;
+    position: absolute;
+    top: 70%;
+    left: 50%;
```
Comment (wax-bear) [inline styles/style.css:66]: Please mind the indentation here.

File: styles/style.css [hunk 4]
```diff
@@ -6,3 +6,183 @@ Green: #00B172
...
+  transform: translate(-50%, -50%);
```
Comment (wax-bear) [inline styles/style.css:67]: What is the purpose of this key-value pair?

File: styles/style.css [hunk 5]
```diff
@@ -6,3 +6,183 @@ Green: #00B172
...
+    left: 50%;
+    top: 260%;
+    margin: auto;
+}
+#spotify-app-pic{
+    height: 500px;
+    padding: 70px 00px 40px 50px;
+}
+
+/*
```
Comment (wax-bear) [inline styles/style.css:180]: What is the function of this here? :)

**Review by wax-bear — COMMENTED** [review]
Great stuff Maya!

Looking at your website, there are a few aspects that I believe are deviating from the original as shown in the LAB. For example, the styling of the nav bar items, and the white space around the green bottom pannel. You could compare the original to your clone and then iterate towards a more definite clone.

Please check out my comments in the HTML and CSS as well.

File: index.html (hunk 1 above)
Comment (MayaAusset) [reply index.html:19]: I forgot to delete them, it is for just for clarity while I write the code

File: styles/style.css (hunk 4 above)
Comment (MayaAusset) [reply styles/style.css:67]: It was actually a test, thought I deleted it , should have reviewed my code better sorry

File: styles/style.css (hunk 5 above)
Comment (MayaAusset) [reply styles/style.css:180]: A test as well which I forgot to delete. Next time I will review my code better.
Thank you very much for the feedback !

**Pull request closed without merging**